lib/sync/index.js

```diff
@@ -89,38 +106,73 @@ function setLogLevel(dataset_id, params, cb) {
   }
 }
 
-function sync(dataset_id, params, callback) {
+function sync(dataset_id, params, cb) {
   syncUtil.doLog(dataset_id, 'info', 'sync');
 
   // TODO
 
   return cb(null, {});
 }
 
-/* Synchronise the individual records for a dataset */
-function syncRecords(dataset_id, params, callback) {
+// Synchronise the individual records for a dataset
+function syncRecords(dataset_id, params, cb) {
   syncUtil.doLog(dataset_id, 'info', 'syncRecords');
 
   // TODO
 
   return cb(null, {});
 }
 
+
+/**
+ * Connect to mongodb & redis with the given connection urls.
+ * This should be called before `start()` is called
+ * 
+ * @param {string} mongoDBConnectionUrl A MongoDB connection uril in a format supported by the `mongodb` module
+ * @param {string} redisUrl A redis connection url in a format supported by the `redis` module
+ * @param {function} cb 
+ */
+function connect(mongoDBConnectionUrl, redisUrl, cb) {
+  if (arguments.length < 3) throw new Error('connect requires 3 arguments');
+  
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)